src/Microsoft.TestPlatform.TestHostProvider/Hosting/DotnetTestHostManager.cs

```diff
@@ -44,6 +44,7 @@ public class DotnetTestHostManager : ITestRuntimeProvider
     {
         private const string DotnetTestHostUri = "HostProvider://DotnetTestHost";
         private const string DotnetTestHostFriendltName = "DotnetTestHost";
+        private const string TestAdapterRegexPattern = @".*.TestAdapter.dll";
```

Comment:
Since this is publicly documented, should we pick it up from OM? Please note it may require an update in vsualm.